examples/chrono/chrono.cpp

```diff
@@ -0,0 +1,150 @@
+﻿/*
+ * Copyright (c) 2022, Christopher Kormanyos
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <chrono>
+
+#include <avr/interrupt.h>
+#include <avr/io.h>
+
+static void app_hw_init();
+
+auto main() -> int
+{
+  // Initialize the application hardware. This includes WDT, PORTB.5 and TIMER0.
+  app_hw_init();
+
+  for(;;)
+  {
+    // Toggle the LED on portb.5.
```

Comment:
> ... uses tabs

Oh, here we have our first actual conflict of style. I do not usually use tabs.

But..., it's your project, your style. I'll switch to tabs if that's the style here. No problem.

> add a `.clang-format` file and enforce it using CI?

That would be nice. I also recently started to use `clang-tidy` and I am _very_ happy with the results. I have starte in my own projects to add tidy checks to CI as well.

I think, however, that `clang-format` and `clang-tidy` enforce/check/CI should be undertaken in qa second step, as i'd like to get the `<chrono>` example in there and finally get started/to-work/finished with #10.

After that, I'd actually recommend to start with `clang-tidy`, then move on to format issues.

In the intermediate time, I'll go ahead and switch over to tabs. 

Are tabs the consensus around here?

Cc: @salkinium and @chris-durand

Reply:
> Oh, here we have our first actual conflict of style. I do not usually use tabs.
>
> But..., it's your project, your style. I'll switch to tabs if that's the style here. No problem.

Tabs is what we use in the modm project because we inherited that from its predecessor xpcc.

Reply:
:+1: for automatic formatting and static analysis

Reply:
I would prefer spaces over tabs, but I don't want to have a confusing mix of policies on modm, so we're going with tabs for now. clang-format makes it easy to work with anyways.

Reply:
If I am here, I will comply. Thx for clarification.